static/js/components/VideoCard_test.js

```diff
@@ -46,16 +53,34 @@ describe('VideoCard', () => {
     it(`video controls ${expect(shouldShow)} be shown for ${testDescriptor}`, () => {
       let isAdmin = adminPermissionSetting;
       let wrapper = renderComponent({isAdmin: isAdmin});
-      assert.equal(wrapper.find(".actions a.edit-link").exists(), shouldShow);
+      //wrapper.find(".actions").children().at(0).children.at(0).children.at(0).children()
+      let menuItems = wrapper.find("Menu").props().menuItems;
+      assert.equal(menuItems.length, (shouldShow ? 3 : 1));
+      if (menuItems.length === 3) {
+        assert.equal(menuItems[1].label, 'Edit');
+        assert.equal(menuItems[2].label, 'Save To Dropbox');
+      }
     });
   });
 
-  it('handles an edit button and share button click', () => {
+  it('handles an edit button, download button, and share button click', () => {
```

Comment:
~~download button~~ -> `"save to dropbox" button` et al

Reply:
OK